Ladybird/Headless/Test.cpp

```diff
@@ -70,7 +70,7 @@ static ErrorOr<void> collect_dump_tests(Vector<Test>& tests, StringView path, St
             continue;
         }
 
-        if (!name.ends_with(".html"sv) && !name.ends_with(".svg"sv) && !name.ends_with(".xhtml"sv))
+        if (!name.ends_with(".html"sv) && !name.ends_with(".svg"sv) && !name.ends_with(".xhtml"sv) && !name.ends_with(".xht"sv))
```

Comment:
Alternative thought: Move the asset files into `data` and remove this check entirely?

Reply:
Doing that would involve a fair amount of renaming, since you'd need to move `include.js` to the data directory. I've done a quick count and there are currently 120 files that would need to be moved and their paths updated. The WPT import script would also need to be updated to put all the support files in the correct place.

If we're going to do something like this, maybe we should do the same thing as the WPT runner, which excludes all files from any folder called "resources", "support", or "tools" and also excludes all files from the root "common" folder: https://github.com/web-platform-tests/wpt/blob/caa5c5bb7d86612b0f3f291b208d4189bb521697/tools/manifest/sourcefile.py#L181-L185

(WPT also treats certain filenames as support files, see: https://github.com/web-platform-tests/wpt/blob/caa5c5bb7d86612b0f3f291b208d4189bb521697/tools/manifest/sourcefile.py#L312-L322)

Reply:
Fair enough! I forgot about WPT.